tests/test_web_protocol.py

```diff
@@ -237,7 +237,7 @@ def test_srv_keep_alive_disable(srv) -> None:
     srv.data_received(
         b'GET / HTTP/1.1\r\n\r\n')
 
-    await asyncio.sleep(0)
+    await asyncio.sleep(0.01)
```

Comment:
Does it worth to have some helper function which does `await asyncio.sleep` with proper time? To not change a lot of constants everywhere all the time?

Reply:
I think all 5-6 places in the single test file are fixed.
Don't want to add a helper, at least now.